src/networking.c

```diff
@@ -28,7 +28,7 @@ int postponeClientRead(client *c);
 char *getClientSockname(client *c);
 int ProcessingEventsWhileBlocked = 0; /* See processEventsWhileBlocked(). */
 __thread sds thread_shared_qb = NULL;
-__thread int thread_shared_qb_used = 0; /* Avoid multiple clients using shared query
+__thread int thread_shared_qb_used = 0; /* Avoid multiple clients using reusable query
```

Comment:
you renamed everything, but didn't rename the variable

Reply:
🙃i only updated the commands in the comments and the `resetReusableQueryBuf()` method (I added), so it looks like almost everything has changed.

Reply:
ok, it looked like you changed a lot of lines for that.. (could cause a lot of conflict).
i thought i'm suggesting just editing one comment.
but i didn't look at the original code. do what you think is best.

Reply:
i actually reworked a pretty big chunk of code, the code is now much more efficient and readable.

Reply:
ok, so feel free to rename the variable too